src/Expecto.VisualStudio.TestAdapter/TestCases.fs

```diff
@@ -0,0 +1,18 @@
+﻿module TestCases
+
+open System.Linq
+open Microsoft.VisualStudio.TestPlatform.ObjectModel
+
+let fullyQualified (name:string) =
+    //Expecto uses / instead of . as separator
```

Comment:
Instead of adding a comment in the code, could you add a test case with a meaningful name, please :)?

Reply:
Something like this?
```suggestion
testCase "TestCases are fully qualified with dots from slashes" <| fun _ ->
    let name = "tests/hello/world"
    let case = TestCases.testCase "source" name
    Expect.equal case.FullyQualifiedName "tests.hello.world" "Should use '.' as separator"
```
I'm not sure where you want it; it doesn't really seem like it belongs in the samples.

Reply:
It could be placed in a separate project instead of samples. But yes this is what I mean :)

Reply:
Ok hows this? I've just added some unit tests in a new file.

Organizing or creating a project for unit testing the `TestAdapter` other than the samples could probably be it's own issue.